### PR TITLE
fix(amplify): upgrade uuid

### PR DESCRIPTION
**Pull request by fjnoyp**
*Issue #, if available:* #434

*Description of changes:*

Upgrade UUID dependency and remove conflicting flutter driver code 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: packages/amplify_auth_cognito/example/test_driver/app.dart [hunk 1]
```diff
@@ -9,3 +10,4 @@ void main() {
   // any widget you are interested in testing.
   app.main();
 }
+ */
```
Comment (Amplifiyer) [inline packages/amplify_auth_cognito/example/test_driver/app.dart:13]: Do we need this file at all?

Comment (fjnoyp) [reply packages/amplify_auth_cognito/example/test_driver/app.dart:13]: @haverchuck what are your thoughts - can I delete these?

Comment (fjnoyp) [reply packages/amplify_auth_cognito/example/test_driver/app.dart:13]: Confirmed with Dustin that we can delete these.

**Review by haverchuck — APPROVED** [review]

File: packages/amplify_datastore/pubspec.yaml [hunk 2]
```diff
@@ -14,7 +14,7 @@ dependencies:
   amplify_core: 0.1.0
   plugin_platform_interface: ^1.0.1
   date_time_format: ^1.1.0
-  uuid: ^2.2.2
```
Comment (Ashish-Nanda) [inline packages/amplify_datastore/pubspec.yaml:17]: Is the `uuid` dependency needed in the `amplify_datastore` package when its already a dependency in the `amplify_datastore_plugin_interface` package?

Comment (fjnoyp) [reply packages/amplify_datastore/pubspec.yaml:17]: Thanks for the comment.  It’s a good point, the datastore package doesn’t need uuid.  I just tested and validated removing it won’t cause any problems.  I’ll remove the package.

**Review by Ashish-Nanda — APPROVED** [review]
Looks good 🚀

**Pull request merged**